### ISSUE TITLE
Issue with connecting via SSL to postgres

### ISSUE THREAD

**Issue by simonv3** [TheTorProject/ooni-api#1]
I'm not 100% sure how to set up postgresql to allow connections via SSL, or how to make this work, but whenever I ran `grunt serve` the Datasource.prototype.setup function in `node_modules/loopback-datasource-juggler/lib/datasource.js` would print out:

```
Running "run:development" (run) task
Browse your REST API at http://localhost:3000/explorer
Web server listening at: http://localhost:3000/
Connection fails:  [Error: The server does not support SSL connections] 
It will be retried for the next request.
Fatal error: The server does not support SSL connections
```

And would then quit before getting to the watch command. I think this is the output of the concurrent `server` command in the grunt file, which periodically would throw this after getting to `watch`, but then everything would be fine and the server would continue running. This only happened once in a blue moon though. 

I've traced this to the fact that my postgresql install doesn't allow SSL connections, so what I tried setting SSL to false, but it turns out that node-postgres (which loopback uses) ignores ssl=false connections (https://github.com/brianc/node-postgres/issues/848). When adding the fix in the node-postgres dependency suggested in that issue, connecting works fine. 

I've also investigated whether my postgres (which I installed using the postgres app you link to, but a long long while ago) would support SSL, and I think it would `$ /usr/local/bin/postgres -c ssl=on` spits out: 
```
postgres does not know where to find the server configuration file.
You must specify the --config-file or -D invocation option or set the PGDATA environment variable.
```
which is different from what it would say if it wasn't supported (as far as I can gather: http://stackoverflow.com/questions/32634763/how-to-check-if-postgresql-installation-supports-ssl/32643456#32643456).

So, TL;DR - either I need to find instructions on how to set up postgres so that it supports SSL (any ideas?) and that needs to be specified in the readme, or we should submit a patch to node-postgres so that SSL isn't necessary.

**Comment by hellais** [comment]
Can you try using what is in PR #2 and see if setting the SSL false to either staging or production will work properly. In the past we had a bit over-engineered the datasources loading mechanism and I believe that lead to some bugs.

**Comment by hellais** [comment]
This has been fixed since a bit.
